topcoffea/modules/corrections.py

```diff
@@ -58,8 +58,6 @@ def get_method1a_wgt_doublewp(effA, effB, sfA, sfB, cutA, cutB, cutC):
 # Evaluate btag sf from central correctionlib json
 def btag_sf_eval(jet_collection,wp,year,method,syst):
```

Comment:
@MatthewDittrich shall we add a function argument, with a default value equal to `True`, to give us the opportunity to set the SFs to 1:

Something like
```
if not apply_sf:
   sf = ak.ones_like(jet_collection.pt)
else:
   [usual code]
```
I would like to verify afterwards the application of btag SFs for our Run3 ttX analysis

Reply:
Apart from this, everything else looks fine to me, thanks!

Reply:
I agree this would be one possible way of allowing the b tagging SFs to be turned off. However, to achieve the same end, I wonder if it might be more straightforward to simply not apply the btag SFs in the processor (i.e., just add an if statement to not apply them for whichever years you currently do not want to apply them for)?

Reply:
@kmohrman you are right, it's a cleaner way to do that